packages/cli/src/plugins/react.ts

```diff
@@ -104,9 +104,11 @@ export function react(config: ReactConfig = {}): ReactResult {
             if (Object.keys(contract.address).length > 1) {
               innerHookParams.address = `${contract.meta.addressName}[chainId as keyof typeof ${contract.meta.addressName}]`
               imports.add('useNetwork')
+              imports.add('useChainId')
               innerContent = dedent`
                 const { chain } = useNetwork()
-                const chainId = config.chainId ?? chain?.id
```

Comment:
if chainId is falsy, then the ABI lookup that follows this line will return undefined, and the contract read won't execute.

This is inconsistent with wagmi's `useContractRead` default behavior, where the public client's chain ID is used (using `useChainId` hook) when none given.